multicast/__init__.py

```
@@ -365,6 +426,10 @@
 if socket.__name__ is None:
 	raise ModuleNotFoundError("FAIL: we could not import socket. ABORT.") from None
 else:  # pragma: no branch
+	# skipcq: PYL-W1203
+	logging.getLogger(__module__).debug(
+		f"Setting default packet timeout to {_MCAST_DEFAULT_TTL:n}",  # skipcq: PYL-W1203 - see PEP-3101
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)